packages/core/src/client.ts

```diff
@@ -3344,6 +3348,10 @@ export class MedplumClient extends TypedEventTarget<MedplumClientEventMap> {
 
   private async pollStatus<T>(statusUrl: string, options: MedplumRequestOptions, state: RequestState): Promise<T> {
     const statusOptions: MedplumRequestOptions = { ...options, method: 'GET', body: undefined, redirect: 'follow' };
+    if (options?.asyncReqCancelSignal?.aborted) {
+      await this.request('DELETE', statusUrl, { ...options, method: 'DELETE', body: undefined });
```

Comment:
The `delete()` and `request()` methods will correctly set the `method`

```suggestion
      await this.delete(statusUrl, options);
```

Reply:
Still need to explicitly remove the `body`, but otherwise can remove the explicit method